### DISCUSSION TITLE
Outdated semver filter

### DISCUSSION THREAD

**Discussion by lomby92** [npm/feedback#308] [CLI] [answered]
Should the **npm outdated** command have a filter to specify if you just want to list packages with update for major, minor or patch?

In my opinion this filter should accept a list of values. For example, as a developer, i could want to filter only major and minor fixes.

E.G.: `npm outdated --include-versions major,minor`

**Comment by MylesBorins** [comment, marked as answer]
Hey @lomby92 thanks for dropping by. If you want to suggest a specific new feature to a CLI command the best way to do it is by opening an issue over at https://github.com/npm/rfcs and follow the issue template. All opened issues are triaged by the team!

**Reply by lomby92** [reply]
Thanks, but maybe the right repo is: https://github.com/npm/rfcs 😄
